### PR TITLE
Add blocks API to monitor dashboard

### LINKED ISSUE #4651: Add missing blocks endpoints in monitor dashboard
### Problem

The monitor dashboard has not been updated properly whenever we add new endpoints.

Split adding endpoints for `blocks` endpoints out of [Ticket 4050](https://github.com/hashgraph/hedera-mirror-node/issues/4050) into this new subticket, so that it can be completed this sprint.

### Solution

- Add new checks for every `blocks` endpoint that is missing
- Add properties to control enablement of checks

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by MarcKriguerAtHedera**
**Description**:

* Add blocks API to monitor dashboard
* Add properties to control blocks API check

**Related issue(s)**:

Fixes #4651
Part of #4050

**Notes for reviewer**:
When running the API monitor, two additional tests now appear, for Resource "block":

- Successfully retrieved block from with 1000 seconds ago (the Freshness test)
- Successfully called blocks for single block

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/monitoring/monitor_apis/config/default.serverlist.json [hunk 1]
```diff
@@ -24,6 +24,12 @@
     "intervalMultiplier": 1,
     "limit": 10
   },
+  "block": {
+    "enabled": true,
+    "freshnessThreshold": 1000,
```
[the remark this answers is not in the document]
Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/monitoring/monitor_apis/config/default.serverlist.json:29]: Done.

File: hedera-mirror-rest/monitoring/monitor_apis/block_tests.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+const mandatoryParams = [
+  'count',
+  'gas_used',
+  'hapi_version',
+  'hash',
+  'name',
+  'number',
+  'previous_hash',
+  'size',
+  'timestamp',
```
[the remark this answers is not in the document]
Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/monitoring/monitor_apis/block_tests.js:50]: Done.

File: hedera-mirror-rest/monitoring/monitor_apis/block_tests.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+const blocksPath = '/blocks';
+const resource = 'block';
+const resourceLimit = config[resource].limit || DEFAULT_LIMIT;
+const jsonRespKey = 'blocks';
+const mandatoryParams = [
+  'count',
+  'gas_used',
+  'hapi_version',
+  'hash',
+  'name',
```
[the remark this answers is not in the document]
Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/monitoring/monitor_apis/block_tests.js:46]: Added it.  Good catch!

File: hedera-mirror-rest/monitoring/monitor_apis/config/default.serverlist.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
   "port": 3000,
   "servers": [
     {
-      "baseUrl": "http://127.0.0.1:5551",
-      "name": "local"
+      "baseUrl": "https://testnet.mirrornode.hedera.com",
+      "name": "testnet"
```
Comment (steven-sheehy) [inline hedera-mirror-rest/monitoring/monitor_apis/config/default.serverlist.json:6]: undo

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/monitoring/monitor_apis/config/default.serverlist.json (hunk 4 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/monitoring/monitor_apis/config/default.serverlist.json]: Undone.  That lives only in my "serverlist.json" file now (not checked in).

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**